addons/addon-base-ui/packages/base-ui/src/models/authentication/AuthenticationProviderConfigsStore.js

```diff
@@ -74,22 +68,6 @@ const AuthenticationProviderConfigsStore = BaseStore.named('AuthenticationProvid
       });
     },
 
-    getUpdateAuthenticationProviderConfigEditor(authenticationProviderConfigId) {
```

Comment:
This method is being called by `addons/addon-base-ui/packages/base-ui/src/parts/authentication-providers/EditAuthenticationProvider.js` (line 95). Do we not need changes there?

Reply:
Good catch! Will update.